scripts/codeCoverage.sh

```diff
@@ -1,69 +1,73 @@
 #!/bin/bash
 LAUNCH_DIR=`pwd`
+PROJECT="FileIO"
+OBJECT_DIR="${PROJECT}.dir"
 
-# If you need to add a header to the code coverage please put it here
-# example "myfile.h myotherfile.h"
+# --- WHITELIST ---
+# Add a header to the code coverage here
+#    Examples:
+#    HEADER_WHITELIST="myfile.h"
+#    HEADER_WHITELIST="myfile.h myotherfile.h"
 HEADER_WHITELIST=
 
-# black listing of files can also be done
-# please see commented out execution of gcovr at the bottom
-# example: 
-#  SOURCE_BLACKLIST=".*ProbeTransmogrifier.cpp.*"
-
+# --- BLACKLIST ---
+# Blacklisting of files can also be done
+#    Please see commented out execution of gcovr at the bottom.
+#    Example: 
+#       SOURCE_BLACKLIST=".*ProbeTransmogrifier.cpp.*"
+SOURCE_BLACKLIST="/usr/local/probe/*"
```

Comment:
You are right - it is pure regex not wildcard. I have made this change in another file and will update this PR today with the correct blacklist regex.